sample_project/Assets/SampleViewer/Samples/FeatureLayer/FeatureData.cs

```diff
@@ -24,11 +21,11 @@ public class FeatureData : MonoBehaviour
     private void Start()
     {
         cameraLocationComponent = ArcGISCamera.GetComponent<ArcGISLocationComponent>();
+        locationComponent = transform.GetComponent<ArcGISLocationComponent>();
         featureHP = transform.GetComponent<HPTransform>();
-        cameraHP = ArcGISCamera.GetComponent<HPTransform>();
         featureHP = transform.GetComponent<HPTransform>();
```

Comment:
```suggestion
```
Nitpick: Not part of your changes, but a bit of clean-up . 
`private float distance;` can be removed too

Reply:
We're going to do a much larger PR cleaning up all the scripts in the near future. There is a lot that needs to be changed.